src/components/DelegatesTable/DelegateVoteCheckbox.vue

```diff
@@ -68,6 +88,12 @@ export default {
     font-size: 24px !important;
     height: 24px !important;
   }
+  &__upvote {
+    color: map-get($adm-colors, 'good');
+  }
+  &__downvote {
```

Comment:
&__upvote and &__downvote selectors serve as modifiers of the &__icon element, so you must name them correctly

Reply:
Check BEM Methodology https://en.bem.info/methodology/quick-start/#modifier

Reply:
Name them like this:
&__icon--good
&__icon--danger